pkg/model/constants.go

```diff
@@ -54,4 +54,7 @@ const (
 	MaxUnavailableNumberOfPods            = 1
 	ServiceMonitorName                    = ApplicationName + "-service-monitor"
 	MigrateBackupName                     = "migrate-backup"
+	DatabaseSecretSslModeProperty         = "SSLMODE"
+	DatabaseSecretSslCert                 = ApplicationName + "-db-ssl-cert-secret"
+	DatabaseConnectionParamsProperty      = "DB_CONN_PROPERTY"
```

Comment:
While this is valid for RH-SSO, please note that this property has a different name in Keycloak image (`JDBC_PARAMS`), see the [docs](https://github.com/keycloak/keycloak-containers/blob/master/server/README.md#specify-jdbc-parameters).

Reply:
:+1: